classes/Page/Page.mjs

```diff
@@ -0,0 +1,135 @@
+export class Page {
+    constructor(page = {}) {
+        this.page = page
+        if(!this.page.id) {
+            this.page.id = Date.now().toString();  // ticket to replace with MongoDB ObjectID()
+        }
+    }
+    
+    get id() {
+        return this.page.id
+    }
+
+    set id(id) {
+        this.page.id = id
+    }
+
+    // Method to represent the page as a canvas with annotation page layer
+    asCanvas() {
+        return {
+            type: 'Canvas',
+            '@context': 'http://iiif.io/api/presentation/3/context.json',
+            id:this.page.id ?? 0,
+            height: this.page.height ?? 0,
+            width: this.page.width ?? 0,
+            label: this.page.label ?? '',
+            summary: this.page.summary ?? '',
+            thumbnail: this.page.thumbnail ?? '',
+            navDate:this.page.navDate ?? '',
+            placeHolderCanvas:this.page.placeHolderCanvas ?? '',
+            accompanyingCanvas:this.accompanyingCanvas ?? '',
+            duration:this.duration ?? '',
+            start:this.start ?? '',
+            rendering:this.rendering ?? '',
+            items:this.items ?? [],
+        };
+    }
+
+    // Method to create a new page
+    create() {
+        return Promise.resolve(new Page())
+    }
+
+    // Method to remove a page
+    remove() {
+        return Promise.resolve()
+    }
+    
+    // Method to save changes to a page
+    save() {
+        return Promise.resolve()
+    }
+    
+    // Method to fetch a page
+    fetch() {
+        return Promise.resolve(new Page())
+    }
+
+    // Method to add membership reference to manifest
+    addMembershipToManifest() {
```

Comment:
this was unclear in the docs, I know. This is to get the Manifest container object this Page is in/partOf

Reply:
updated , resolved.